packages/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -199,22 +199,7 @@ const Encryption = WebexPlugin.extend({
       return Promise.reject(new Error('Cannot encrypt `scr` without first setting `loc`'));
     }
 
-    // first we get the scr json, then we create an SCR instance using the key json and then we create a JWE using the key jwk
-    return this.getKey(key, options).then((k) => {
-      if (!k?.jwk) {
```

Comment:
Isn't it necessary to still do this validation and throw error?

Reply:
Had 1:1 that this is just a revert to the original code before Encryption PR merge. Resolving